api/index.js

```diff
@@ -0,0 +1,5 @@
+const healthcheck = require('./healthcheck');
+const dc = require('./dc');
+const marvel = require('./marvel');
+
```

Comment:
Instead of concatenating the routes like this - destructuring!


```
const healthcheck = require('./healthcheck');
const dc = require('./dc');
const marvel = require('./marvel');

const [healthcheck, marvel, dc] = routes;

module.exports = routes;
```